roles/repository/molecule/default/tests/test_repository.py

```diff
@@ -186,6 +203,24 @@ def test_newly_added_properties_are_set(host):
     assert_that(b'index.subsystem.name=noindex' in content)
     assert_that(host.socket("tcp://0.0.0.0:1121").is_listening)
 
+
 def test_no_ghostscript(host):
     host.run_expect([127], "gs")
     host.run_expect([127], "ghostscript")
+
+
+def test_share_CSRF_config(host):
+    """Check basic CSRF renderinng in share XML file"""
+    alfresco_cfg_dir = '/etc/opt/alfresco/content-services'
+    share_cfg_dir = 'classpath/alfresco/web-extension'
+    share_cfg_file = '/'.join([alfresco_cfg_dir,
+                              share_cfg_dir,
+                              'share-config-custom.xml'])
+    xmlpath = './config[@condition="CSRFPolicy"]/properties'
+    with host.sudo():
+        share_cfg = host.file(share_cfg_file).content_string
+    share_xml = xmlreader.fromstring(share_cfg)
+    referer = share_xml.findall('{}/referer'.format(xmlpath))
+    origin = share_xml.findall('{}/origin'.format(xmlpath))
+    assert_that(referer[0].text == 'http://localhost:8080/share/.*')
+    assert_that(origin[0].text == 'http://localhost:8080')
```

Comment:
can you add an additional url in the moelcule test configuration so we can assert that also the separators are handled correctly (since they were the actual bug for this issue)